### PR TITLE
Ews v2 Fix None obj error

### PR DESCRIPTION
**Pull request by ostolero**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-26203)
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-26278

## Description
Bug fix for None objects arriving in our response and failing the parse. this matches the old version behavior.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2_test.py [hunk 1]
```diff
@@ -494,6 +494,33 @@ def test_list_parse_item_as_dict():
     assert len(effetive_right_res) == 2
 
 
+def test_parse_item_as_dict_with_empty_field():
```
Comment (JudahSchwartz) [inline Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2_test.py:497]: lets add a test that False/empty string does  appear

File: Packs/MicrosoftExchangeOnPremise/ReleaseNotes/2_0_3.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### EWS v2
+
+- Fixed an issue where items that arrived with an empty field failed to be parsed.
```
Comment (JudahSchwartz) [inline Packs/MicrosoftExchangeOnPremise/ReleaseNotes/2_0_3.md:6]: would write something more user friendly
Fixed an issue when returning drafts

**Review by JudahSchwartz — APPROVED** [review]
nice find
